### DISCUSSION TITLE
Rate Change Selection (Mobile)

### DISCUSSION THREAD

**Discussion by jundalis** [ppy/osu#26153] [Ideas] [answered]
For rate change selection on mobile specifically, it’s difficult to choose a specific rate, as you can’t really see the number under your finger and it’s hard to get on a specific rate with the drag feature. So I think there should be a box where people could type the exact rate they want because it’s hard to get it exact especially on mobile.

**Comment by Gabixel** [comment, marked as answer]
You're probably referring to something like https://github.com/ppy/osu/discussions/24900, right? (Spoiler: it's planned)

**Reply by jundalis** [reply]
Oh I’m sorry I didn’t see that

**Reply by Gabixel** [reply]
No worries. You can mark this as resolved since it's a duplicate
